internal/utils/config.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import "github.com/spf13/viper"
+
+type Config struct {
+	DbDriver         string `mapstructure:"DB_DRIVER"`
+	DbSource         string `mapstructure:"DB_SOURCE"`
+	PostgresUser     string `mapstructure:"POSTGRES_USER"`
+	PostgresPassword string `mapstructure:"POSTGRES_PASSWORD"`
+	PostgresDb       string `mapstructure:"DB_NAME"`
+	ServerAddress    string `mapstructure:"SERVER_ADDRESS"`
+}
```

Comment:
I don't know if it would be possible, but it is better to group those values in a structure related to the domain in which they are being used.

```go
type Config struct {
	PostgresDbConfig DB
	ServerAddress             string `mapstructure:"SERVER_ADDRESS"`
}

type DB struct {
        DbDriver                string `mapstructure:"DB_DRIVER"`
	DbSource               string `mapstructure:"DB_SOURCE"`
	PostgresUser         string `mapstructure:"POSTGRES_USER"`
	PostgresPassword string `mapstructure:"POSTGRES_PASSWORD"`
	PostgresDb            string `mapstructure:"DB_NAME"`
}
```

Reply:
I tried to break them into different structs but had no success. 
Could not find a solution to make it work neither